modules/nw-ovn-kubernetes-live-migration-about.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+[id="nw-ovn-kubernetes-live-migration-about_{context}"]
+= Live migration to the OVN-Kubernetes network plugin overview
+
+The live migration method is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources, are migrated to the OVN-Kubernetes network plugin without service interruption. It is available for {product-title}, {product-dedicated}, {product-rosa}, and Azure Red Hat OpenShift deployment types It is not available for HyperShift deployment types. This migration method is valuable for deployment types that require constant service availability and offers the following benefits:
+
+* Continuous service availability
+* Minimized downtime
+* Automatic node rebooting
+* Seamless transition from the OpenShift SDN network plugin to the OVN-Kubernetes network plugin.
+
+Although a rollback procedure is provided, the live migration is intended to be a one-way process.
+
+////
+[IMPORTANT]
+====
+Migrating to or from the OVN-Kubernetes network plugin is not supported for managed OpenShift cloud services such as {product-dedicated}, Azure Red Hat OpenShift(ARO), and Red Hat OpenShift Service on AWS (ROSA).
+====
+////
+
+include::snippets/sdn-deprecation-statement.adoc[]
+
+The following sections provide more information about the live migration method.
+
+[id="supported-platforms-live-migrating-ovn-kubernetes"]
+== Supported platforms when using the live migration method
+
+The following table provides information about the supported platforms for the live migration type.
+
+.Supported platforms for the live migration method
+[cols="1,1", options="header"]
+|===
+| Platform              | Live Migration
+
+| Bare metal hardware             |&#10003;
+| Amazon Web Services (AWS)       |&#10003;
+| Google Cloud Platform (GCP)     |&#10003;
+| {ibm-cloud-name}                |&#10003;
+| Microsoft Azure                 |&#10003;
+| {rh-openstack-first}            |&#10003;
+| VMware vSphere                  |&#10003;
+| AliCloud                        |&#10003;
+| Nutanix                         |&#10003;
+|===
+
+[id="considerations-live-migrating-ovn-kubernetes-network-provider_{context}"]
+== Considerations for live migration to the OVN-Kubernetes network plugin
+
+Cluster administrators should consider the following before using the live migration method to the OVN-Kubernetes network plugin:
+
+* The live migration procedure is unsupported for clusters with OpenShift SDN multitenant mode enabled. You must disable multitenant mode before beginning the process.
```

Comment:
For multitenant mode cluster, I don't think it can be disalbed. so we just say it's unsupported.

```suggestion
* The live migration procedure is unsupported for clusters with OpenShift SDN multitenant mode enabled. 
```